docs/querying/sql-functions.md

```diff
@@ -1077,11 +1077,42 @@ Calculates the base-10 of the numeric expression.
 
 ## LOOKUP
 
-`LOOKUP(<CHARACTER>, <CHARACTER>[, <CHARACTER>])`
+Look up `expr` in a registered [query-time lookup table](lookups.md) named `lookupName`. Returns `replaceMissingValueWith`, which defaults to `null`, when the `expr` is null or when the lookup does not contain a value for `expr`.
 
-**Function type:** [Scalar, string](sql-scalar.md#string-functions)
+* **Syntax:** `LOOKUP(expr, lookupName[, replaceMissingValueWith])`
+* **Function type:** Scalar, string
+
+<details><summary>Example</summary>
+
+The following example uses a map lookup table, named `acronym_to_name` made with the following key-value pairs:
+
+```json
+{
+	“SJU”: Luis Munoz Marin International Airport”,
+	“IAD”: “Dulles International AirportDulles International Airport”
+}
+```
+
+The example uses `acronym_to_name` to map the `Origin` column from the `flight-carriers` datasource to its full airport name. Returns `NaN` if no matching value exist in the lookup table.
```

Comment:
Is `NaN` a good default value to use, when each lookup key-value pair is already not a number?

Reply:
In retrospect probably non, I could change it to "NULL"